src/frontend/packages/semantic-data-provider/src/dataProvider/dataProvider.js

```diff
@@ -14,6 +14,7 @@ import fetchUserConfig from './utils/fetchUserConfig';
 import fetchVoidEndpoints from './utils/fetchVoidEndpoints';
 import getServerKeyFromType from './utils/getServerKeyFromType';
 import httpClient from './httpClient';
+import handleFiles from './utils/handleFiles';
```

Comment:
For code readability, it's better to import the specific function
```suggestion
import { uploadFile } from './utils/handleFiles';
```